code/game/objects/items/holy_weapons.dm

```diff
@@ -411,6 +413,7 @@
 /obj/item/nullrod/scythe
 	icon_state = "scythe1"
 	inhand_icon_state = "scythe1"
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
handled by build_worn_icon() already

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.